scripts/cargo.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# Usage:
+#   CARGO_BUILD_TARGET=<target> cargo.sh <args>...
+#
+# This script wraps `cargo` invocations, and calls either `cargo`
+# or `cross`, as appropriate. It checks the value of the
+# `CARGO_BUILD_TARGET` environment variable, and calls `cargo` if
+# the target matches the host target, and `cross` otherwise.
+#
+# Use of `cargo`/`cross` can be forced by setting the `CARGO`
+# environment variable to `cargo`/`cross`. This is useful for
+# instance to force the use of `cross` in CI even for the host
+# target.
+#
+# When cross is used, the target directory will be appended
+# `cross-<target>` to avoid libc conflicts in host binaries (e.g.,
+# build scripts, proc macros). The computed value of the target
+# directory can be obtained with `cargo.sh --print-target-dir`.
+#
+# Lastly, when using `cross` this scrips sets some configuration
+# to allow running `youki` tests inside the `cross`` container.
+# Please check the comments in this scrips to learm more about that.
+#
+# Limitations:
+#  * You **must** use the `CARGO_BUILD_TARGET` environment variable
```

Comment:
Isn't this overwritten by
https://github.com/containers/youki/pull/2536/files#diff-3d64996115d0ac3b74e36f57ce6b00d6b4982614c41b1ed39eab8ef43b2e9a2bR38 ?

Reply:
Thanks, good catch :-)